consensus/progpow/consensus.go

```diff
@@ -713,3 +715,28 @@ func (progpow *Progpow) FinalizeAndAssemble(chain consensus.ChainHeaderReader, h
 func (progpow *Progpow) NodeLocation() common.Location {
 	return progpow.config.NodeLocation
 }
+
+// AccumulateRewards credits the coinbase of the given block with the mining
+// reward. The total reward consists of the static block reward and rewards for
+// included uncles. The coinbase of each uncle block is also rewarded.
+func (progpow *Progpow) accumulateRewards(config *params.ChainConfig, state *state.StateDB, header *types.WorkObject, logger *log.Logger) {
```

Comment:
The logger here should come from progpow.logger rather than passing it in.

Reply:
ok